### PR TITLE
docs(VISION): fix typo and errors

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**
Fixes typo and errors on the VISION document.

### CODE REVIEW

**Review by edrob999 — CHANGES_REQUESTED** [review]
There are problems with the changes.  I recommend we do the changes in a document

**Comment by orville-wright** [comment]
> There are problems with the changes.  I recommend we do the changes in a document

@edrob999 that's not really the PR process. But, if you feel this needs a special review process we can consider this. 
I recommend that you author the doc you are proposing and discuss it with the Maintainers in the next Maintainers mtg. We can vote on it then.

**Comment by edrob999** [comment]
OK. I'll try inline

File: VISION.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # OpenEBS Umbrella Project Vision
 
 > [!Important]
-> OpenEBS is an "umbrella" project,  composed as a federation of individual sub projects (repositories). The umbrella project, every sub project, repository/repo and file in the OpenEBS organization adopts and follows the same set of umbrella policies in the OpenEBS Community repo. This is the Umbrella Vision for OpenEBS.
+> OpenEBS is an "umbrella" project,  composed as a federation of individual sub-projects (repositories). The umbrella project, every sub-project, repository/repo, and file in the OpenEBS organization adopts and follows the same set of umbrella policies in the OpenEBS Community repo. This is the Umbrella Vision for OpenEBS.
```
Comment (edrob999) [inline VISION.md]: the extra , should be removed ("and" negates need for , in most cases)

File: VISION.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 <BR>
 
 OpenEBS (Open-Elastic-Block-Storage) aims to provide fast, resilient, available, enterprise-grade storage; natively inside a Kubernetes cluster as Hyper-Converged storage. OpenEBS Users provision OpenEBS storage services to Kubernetes applications running natively inside a Kubernetes cluster.
 
-In this document we present the project:
+In this document, we present the project:
```
Comment (edrob999) [inline VISION.md]: The whole phrase should be changed from passive to active tense "This document presents the project::

File: VISION.md [hunk 3]
```diff
@@ -19,22 +19,22 @@ In this document we present the project:
...
 | *Available* | Storage attached to one node can be made addressable to applications running on any node within the cluster through a ```“Nexus” Virtual SAN Fabric``` (i.e. Block mode Storage Area Network). |
-| *Native* | Implemented as ```container native storage```, a Kubernetes natively integrated component, optimized for storage contained inside a Kubernetes cluster (Hyper converged). |
-| *Enterprise grade* | A fully featured, community proven, rigorously tested, trustworthy, Mission Critical product. |
-| *Kubernetes* | OpenEBS focus is for Kubernetes on Linux. |
+| *Native* | Implemented as ```container native storage```, a Kubernetes natively integrated component, optimized for storage contained inside a Kubernetes cluster (Hyper-converged). |
+| *Enterprise grade* | A fully featured, community-proven, rigorously tested, trustworthy, Mission Critical product. |
+| *Kubernetes* | OpenEBS focus is on Kubernetes on Linux. |
```
Comment (edrob999) [inline VISION.md]: should be "for Kubernetes on Linux" not "on"

File: VISION.md [hunk 4]
```diff
@@ -51,25 +51,25 @@ In the past, the OpenEBS project has tried to accomplish too much. Our scope was
 | 9. | ```Rest API``` and ```Cli``` for accessing all capabilities |
 | 10. | Storage Reporting, alerting, monitoring, metrics |
 | 11. | Read/Write Access Modes for RWO, ROX, RWOP - (*RWX with strict limitations) |
-| 12. | Deployable as ```On-premise```, ```SaaS/PaaS in-cloud```, ```Bare Metal```, ```HyperVisor``` infrastructure |
+| 12. | Deployable as ```On-premise``, ```SaaS/PaaS in-cloud``, ```Bare Metal```, ```HyperVisor``` infrastructure |
```
Comment (edrob999) [inline VISION.md]: Hmmmm.  Should probably be "Bare metal, HyperVisor,Hosted/Cloud-based infrastructure"

File: VISION.md [hunk 5]
```diff
@@ -51,25 +51,25 @@ In the past, the OpenEBS project has tried to accomplish too much. Our scope was
...
 <BR>
 
 > **Out-of-Scope**
 
 | Item  | Scope and description     |
 | :---: | :---                      |
 | 1. | Updates to legacy/deprecated storage drivers (archived in the OpenEBS-archive organization) |
-| 2. | Graphical user interface for setup, administration, reporting, dashboards |
+| 2. | Graphical user interface for setup, administration, reporting, and dashboards |
```
Comment (edrob999) [inline VISION.md]: "and" is not needed

File: VISION.md [hunk 6]
```diff
@@ -97,23 +97,23 @@ DataCore, Inc is a CNCF Silver member. DataCore supports CNCF extensively and fu
...
 
 
 ## Archive Organization
 > [!IMPORTANT]
 > The OpenEBS project Leadership/Maintainers also administer the affiliated [**OpenEBS-archive organization**](https://github.com/openebs-archive). <BR>
-> 1. This parallel organization is **OWNED by CNCF** and administrated as a reference archive for OpenEBS ```Legacy``` and ```Deprecated``` OpenEBS repositories, assets, IP, source code, dependencies and resources. <BR>
-> 2. From time to time, we reserve the right to selectively Deprecate and Archive OpenEBS components and  physically migrate them **```out-of```** the OpenEBS Parent project; and into the Archive organization. This process **```officially removes```** those OpenEBS components from the parent OpenEBS project/product. All entities held within the Archive org are **NO LONGER** an active component of the OpenEBS Parent project/product. <BR>
+> 1. This parallel organization is **OWNED by CNCF** and administrated as a reference archive for OpenEBS ```Legacy``` and ```Deprecated``` OpenEBS repositories, assets, IP, source code, dependencies, and resources. <BR>
```
Comment (edrob999) [inline VISION.md]: The comma is not needed (not required when "and" is present)

**Review by orville-wright — APPROVED** [review]
Looks pretty reasonable to me.
- Approved

**Review by niladrih — COMMENTED** [review]
',' and 'and' don't need to sit together. The commas immediately followed by 'and' should have the comma removed.

File: VISION.md (hunk 3 above)
Comment (niladrih) [reply VISION.md]: I suggest "OpenEBS focus is on Kubernetes-on-Linux"

File: VISION.md (hunk 4 above)
Comment (niladrih) [reply VISION.md]: HyperVisor --> Hypervisor

File: VISION.md (hunk 1 above)
Comment (Abhinandan-Purkait) [reply VISION.md]: I used a AI to detect the issues and it seems to suggest a comma. Should I remove it?

File: VISION.md (hunk 2 above)
Comment (Abhinandan-Purkait) [reply VISION.md]: Done.

File: VISION.md (hunk 3 above)
Comment (Abhinandan-Purkait) [reply VISION.md]: Can I get a middle ground on this? How about the below.

`OpenEBS focus is for Kubernetes-on-Linux`

File: VISION.md (hunk 4 above)
Comment (Abhinandan-Purkait) [reply VISION.md]: @edrob999 You want me to remove the terms `On-premise, SaaS/PaaS in-cloud` and change that to only `Bare metal, HyperVisor,Hosted/Cloud-based infrastructure`?

File: VISION.md (hunk 1 above)
Comment (Abhinandan-Purkait) [reply VISION.md]: Done.

File: VISION.md (hunk 5 above)
Comment (Abhinandan-Purkait) [reply VISION.md]: Done.

File: VISION.md (hunk 6 above)
Comment (Abhinandan-Purkait) [reply VISION.md]: Done.

**Comment by niladrih** [comment]
I retract my comments.

**Review by tiagolobocastro — APPROVED** [review]

**Review by avishnu — APPROVED** [review]

**Review by niladrih — APPROVED** [review]

**Pull request merged**